docs/serverless/workers/vllm/environment-variables.md

```diff
@@ -0,0 +1,115 @@
+Ensure to follow the selection note for CUDA 12.1.0 compatibility.
+
+## Environment variables
+
+:::note
+
+`MODEL_NAME` is a mandatory setting.
+All other configurations are optional but recommended for tailored operation.
+
+Boolean values are represented by `0` for false and `1` true.
+:::
+
+### LLM settings
+
+Configure your vLLM Worker by setting environment variables as needed.
+For instance, `MODEL_NAME` might be `openchat/openchat-3.5-1210`, with `HF_TOKEN` set to your Hugging Face API token for accessing private models.
+
+| Name                | Default          | Type/Choices                                              | Description                                                                                                                                                                                                                                                                                               |
+| ------------------- | ---------------- | --------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `MODEL_NAME`        | -                | `str`                                                     | Hugging Face Model Repository (e.g., `openchat/openchat-3.5-1210`).                                                                                                                                                                                                                                       |
+| `MODEL_REVISION`    | `None`           | `str`                                                     | Model revision(branch) to load.                                                                                                                                                                                                                                                                           |
+| `MAX_MODEL_LENGTH`  | Model's maximum  | `int`                                                     | Maximum number of tokens for the engine to handle per request.                                                                                                                                                                                                                                            |
+| `BASE_PATH`         | `/runpod-volume` | `str`                                                     | Storage directory for Huggingface cache and model. Utilizes network storage if attached when pointed at `/runpod-volume`, which will have only one worker download the model once, which all workers will be able to load. If no network volume is present, creates a local directory within each worker. |
```

Comment:
Huggingface -> Hugging Face 

Confusing description. Maybe something like:
When network storage is attached, uses a single Worker to download the model once for all Workers to load. If no network storage is present, creates a local directory within each Worker and downloads the model for each.